src/content/docs/style-guide/writing-docs/article-templates/branching-docs-guide.mdx

```diff
@@ -0,0 +1,254 @@
+* agentConfigFilePath: where the agent yml file lives
+* appInfo: where you define the dropdowns
+    * optionType: value used for overrides, used for query parameters, also used in places for component type appInfo.mdx, appInfoConfigOption, giving a name to the variable used elsewhere on the page 
+    * label: name for the dropdown
+    * placeholder: text that's there before you selected anything
+* options
+    * value: the variable for the name for the tile, hooks into the query param for the URL
+    * displayName: the label of the tile
+    * logo: this is an optional path to an image for the tile
+    * icon: use this if you want to use an icon instead of an image (icon name="fe-server”)
+    * recommendedGuided: Boolean, determines whether guided install content displays (optional)
+
+* steps: steps section defines all the steps on the page, the order these are listed determines the order of the steps
+    * filePath: this is the path to a .mdx file (default file content)
+    * overrides: determines if an .mdx file is swapped in 
+        * skip: Boolean, removes content
+        * selectedOptions: this section hooks into the selected values in the app config to replace content
+            * optionType: this connects to the appInfo variable name
+            * options
+                * value: connects to the specific item in a specific dropdown
+
+
+## Branching MDX files [#mdx-overview]
+
+When working with this install framework, there are specific .mdx files to be aware of.
+
+* Intro
+    * Referenced in the YAML by `introFilePath`
+    * This is the content at the top of the page.
+    * Use for intro content.
+* default 
+    * free mdx file
+* appInfoConfigOption 
+    * this is for dropdowns, use whenever you want dropdowns
+* agentConfig 
+    * this is for the APM agent yml config file
+* appInfo 
+    * this step is the one where you display the content (includes recommendedGuided content)
+
+## A sample MDX file [#sample-mdx]
+
+```
+--- 
+ headingText: Let's begin
+ componentType: default
+ ---
+
+ Boil your water. 
+```
+
+## The appInfo MDX file [#sample-appinfo]
+
+```
+---
+ componentType: appInfoConfig
+ headingText: Tell us about your java app
+ descriptionText: Select how your app is deployed and your framework. The page updates with the instructions you need for your exact Java setup, so you can start monitoring your app’s performance.
+ ---
+
+ ### We recommend the guided install for your app
+
+ A single CLI-command you can run on your system to start monitoring your Java app.
+ Java apps deployed via Tomcat, Wildfly/Jboss, and Jetty (standalone).
+
+ <Button variant="primary">Guided Install</Button>
+
+ If you would prefer to install manually you can follow the steps below.
+```
+
+## The agentConfig MDX file [#sample-agentconfig]
+
+The `agentConfig` MDX file points to the appropriate config file in the install directory and controls what lines in the config file are editable.
```

Comment:
@paperclypse I would change "what" to "which."

Also, how does this file work? Could you extract one or two lines from the example and dissect it for me?

Reply:
@rhetoric101 The final URL is some variation on "docs.newrelic.com/install/[variable]". The variable is set in the agentName field in the YAML file. So, for example, `agentName: java` in the YAML file leads to a URL of "docs.newrelic.com/install/java"